static/app/components/feedback/list/useHasLinkedIssues.tsx

```diff
@@ -23,7 +23,8 @@ export default function useExternalIssueData({group, event, project}: Props) {
   const renderSentryAppIssues = (): ExternalIssueComponent[] => {
     return components
       .map<ExternalIssueComponent | null>(component => {
-        const {sentryApp, error: disabled} = component;
+        const {sentryApp, error} = component;
+        const disabled = typeof error === 'string' ? !error : error;
```

Comment:
```suggestion
        const disabled = typeof error === 'boolean' ? error : !error;
```
Nit

Reply:
is there a reason why we prefer to check for boolean ?